xsd/netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd

```diff
@@ -354,8 +354,43 @@ Rail transport, Roads and Road transport
 	</xsd:complexType>
 	<xsd:element name="Entrance" type="SiteEntrance_VersionStructure" abstract="false" substitutionGroup="SiteComponent">
 		<xsd:annotation>
-			<xsd:documentation>A physical entrance or exit to/from a SITE. May be a door, barrier, gate or other recognizable point of access.</xsd:documentation>
+			<xsd:documentation>A physical entrance or exit to/from a SITE. Maybe a door, barrier, gate or other recognizable point of access.</xsd:documentation>
 		</xsd:annotation>
+		<!-- TODO: Matthias
```

Comment:
Adding all this stuff is no good idea. It is already there in Entrance before. That is why the ambiguity occurs. 
An id is also there: 
![image](https://github.com/NeTEx-CEN/NeTEx/assets/24227470/27f6c07e-df71-45da-bd7a-8570ce5a6be5)

Is the problem that it is not required?

Reply:
What we try to achieve here is that the attribute id gets overridden later with a specific ObjectIdType for that specific element. Theoretically we could add a regular expression then that would guard that the object is named like "codespace:ScheduledStopPoint:something". Not my intention now, but building the relationship between Ref and Id is.